test/sample-test/run_test.sh

```diff
@@ -112,7 +112,9 @@ BASE_DIR=/python/src/github.com/${GITHUB_REPO}
 echo "Clone ML pipeline code in COMMIT SHA ${COMMIT_SHA}..."
 git clone https://github.com/${GITHUB_REPO} ${BASE_DIR}
 cd ${BASE_DIR}
-git checkout ${COMMIT_SHA}
+git config --local user.name 'K8S Bootstrap'
```

Comment:
Would it be possible to put these four lines of code in a single place instead of duplicating them?

Reply:
@IronPan Yang, do you think we can unify those run_test.sh scripts?

Reply:
> Would it be possible to put these four lines of code in a single place instead of duplicating them?

I've looked at how the tests are implemented and it seems a bit complicated to do at this moment. Every test image essentially consists of a base image and just one script file - run_test.sh. No other files are currently included.

Reply:
OK, let's do it later in that case.